### PR TITLE
Migrate to Gradle 5

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```
Comment (avarabyeu) [inline gradlew]: this one is autogenerated. let's not change autogenerated stuff

File: gradlew.bat [hunk 2]
```diff
@@ -14,7 +14,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m"
```
Comment (avarabyeu) [inline gradlew.bat]: this one is autogenerated. let's not change autogenerated stuff

File: gradlew (hunk 1 above)
Comment (valfirst) [reply gradlew]: yes, it was re-generated by wrapper task

Comment (valfirst) [reply gradlew]: https://github.com/gradle/gradle/commit/58e5c9f696d74127d42540e00514881e7491fc9b#diff-3ce07ba0083d748d3a1d38fb9a602643

**Pull request merged**